arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -0,0 +1,930 @@
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceFabric/clusters/{clusterName}": {
+      "patch": {
+        "tags": [
+          "ClusterPatchOperation"
+        ],
+        "x-ms-long-running-operation": true,
+        "operationId": "Clusters_Update",
+        "description": "Update cluster config",
+        "x-ms-examples": {
+          "Patch a cluster": {
+            "$ref": "../examples/ClusterPatchOperation_exmaple.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Resource Group to which the server belongs."
+          },
+          {
+            "name": "clusterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the cluster"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "ClusterUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterUpdateParameters"
+            },
+            "description": "Patch Request"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            }
+          },
+          "202": {
+            "description": "Accept"
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "ClusterGetOperation"
+        ],
+        "operationId": "Clusters_Get",
+        "description": "Get cluster resource",
+        "x-ms-examples": {
+          "Get a cluster": {
+            "$ref": "../examples/ClusterGetOperation_example.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "resourceGroupName",
```

Comment:
this will confuse, it will add the parameter as a property of the client, which doesn't make sense to bind resource group and cluster to the client.